packages/angular-templates/templates-handler.js

```diff
@@ -1,4 +1,4 @@
-angular.module('angular-meteor').config([
+angular.module('angular-templates', []).config([
```

Comment:
@Urigo We're creating new module `angular-templates`. I thought it would be nice if package's name is the same as module's.